RawWaterManagement/CrossSection/schema.json

```diff
@@ -16,12 +16,13 @@
         "position": {
             "type": "number",

```

Comment:
This is not valid json schema
"position": {
"type": "object",
"description": "Property. Description of the relevant position of a cross-section from a reference point",
"properties": {
"order" : {
"type": "number"},
"reference": {
"$ref": "../RawWaterManagement-schema.json#/ngsildRelationship/properties"}
}
}

this is json schema correct. (but maybe it is not what you want)